docs/troubleshooting/migrate_cstor_pools.md

```diff
@@ -0,0 +1,229 @@
+cstor-cspc-zdvk   ip-192-168-29-217   98k        9630M  9630098k  false     1                    1                stripe  ONLINE  4m25s
+```
+- To know more about the cStor volumes resources
+```sh
+$ kubectl get cvc,cv,cvr -n openebs
+NAME                                                                          STATUS   AGE
+cstorvolumeconfig.cstor.openebs.io/pvc-81746e7a-a29d-423b-a048-76edab0b0826   Bound    7m3s
+
+NAME                                                                                           USED   ALLOCATED   STATUS    AGE
+cstorvolumereplica.cstor.openebs.io/pvc-81746e7a-a29d-423b-a048-76edab0b0826-cstor-cspc-4tr5   6K     6K          Healthy   7m3s
+cstorvolumereplica.cstor.openebs.io/pvc-81746e7a-a29d-423b-a048-76edab0b0826-cstor-cspc-xnxx   6K     6K          Healthy   7m3s
+cstorvolumereplica.cstor.openebs.io/pvc-81746e7a-a29d-423b-a048-76edab0b0826-cstor-cspc-zdvk   6K     6K          Healthy   7m3s
+
+NAME                                                                    STATUS    AGE    CAPACITY
+cstorvolume.cstor.openebs.io/pvc-81746e7a-a29d-423b-a048-76edab0b0826   Healthy   7m3s   5Gi
+```
+Here:
+- CVC refers to cStorVolumeManager(Which is responsible for managing cStor volume resources like target service, target deployment, CV and CVR).
+- CV refers to CStorVolume(Which contains the configuration required for process running inside the cStor target pod container).
+- CVR refers to CStorVolumeReplica(Which contains the configuration required to create zfs replica in cStor pool this replica is responsible for reading/writing application IOs to pool. It follows the naming convention like <pv_name>-<pool_name>).
+
+### Lost Node and underlying disk attached to the ndoe
+Performed node and disk lost action as specified [here](#reproduce-the-node-and-underlying-disk-lost-situtation). Now only two nodes are present in the cluster which means whatever data in that corresponding cStor pool was lost. To reconstruct the data we need to make use of the rebuild feature provided by cStor(i.e reconstructing data from other replicas). Below are details after loosing the node and disk
+```sh
+$ kubectl get cspc -n openebs
+NAME         HEALTHYINSTANCES   PROVISIONEDINSTANCES   DESIREDINSTANCES   AGE
+cstor-cspc   2                  3                      3                  20m
+```
+- Pool Manager details
+```sh
+$ kubectl get po -n openebs -l app=cstor-pool
+NAME                               READY   STATUS    RESTARTS   AGE
+cstor-cspc-4tr5-8455898c74-c7vbv   0/3     Pending   0          6m17s
+cstor-cspc-xnxx-765bc8d899-7696q   3/3     Running   0          20m
+cstor-cspc-zdvk-777df487c8-l62sv   3/3     Running   0          20m
+```
+In the above output cStor pool manager i.e **cstor-cspc-4tr5** which was schedule on lost node is in pending state and output of CSPC also shows only two HealthyCSPIInstances.
+
+## Steps to be followed to move the pool from failed node to new node
```

Comment:
I think we can only move volume replica technically. Correct me.

Reply:
Yes